### ISSUE TITLE
mid-2012 Macbook Pro returning "True" for Big Sur

### ISSUE THREAD

**Issue by staze** [laurentpertois/BigSur-Compatibility-Checker#6]
EA for Big Sur is returning True on Mid-2012 Macbook Pro. 

Model:
13-inch MacBook Pro (Mid 2012)
Model Identifier:
MacBookPro9,2

Not sure why, the logic looks correct....

**Comment by laurentpertois** [comment]
Hello @staze 

Sorry for the delay, I had a crazy week.

I found the issue and fixed it. The way I calculate the model is by removing the comma to make it an integer. So 9,2 becomes 92. But during the evaluation, I did not change the minimum number correctly. It was set to 11 which means MacBookPro1,1 as the minimum. It is now set to 110 (MacBookPro 11,0, I know, it does not exist but it's fine).

Thank you very much for bringing it to my attention!

Have a nice day,
